xla/service/gpu/ir_emitter_unnested.cc

```diff
@@ -2557,61 +2558,83 @@ static std::optional<GlobalDeviceId> DeviceConstraint(
   return std::nullopt;
 }
 
-absl::Status IrEmitterUnnested::EmitCopyStartThunk(
-    const HloCopyStartInstruction* instr) {
+absl::Status
+IrEmitterUnnested::EmitCopyStartThunk(const HloCopyStartInstruction *instr) {
   // copy-start has a tuple shape: {host, device, context},
   // or {device, host, context}.
   // Only the destination shape is needed to get the output buffer.
   TF_ASSIGN_OR_RETURN(BufferAllocation::Slice dst_buffer,
                       GetAllocationSliceForHlo(instr,
                                                /*ShapeIndex=*/{0}));
 
-  const HloInstruction* src = instr->operand(0);
-  const Shape& input_shape = src->shape();
+  const HloInstruction *src = instr->operand(0);
+  const Shape &input_shape = src->shape();
```

Comment:
Please separate formatting changes. It's unclear why those are needed.

Reply:
Yes, done.